test/e2e/e2e_test.go

```diff
@@ -236,7 +236,7 @@ func main(m *testing.M) int {
 	}
 
 	// Everything's okay to run tests
-	log.Info("executing e2e tests", "options", globalOpts)
+	log.Info("executing e2e tests")
```

Comment:
I checked the existing Prow jobs and found that this test contains sensitive information, so I disabled printing all parameters to address it

Reply:
if we want to print the params, I will submit a separate pr to make the sensitive params marked as ** by json tags

Reply:
Yes, lets do this.  I noticed this as well on some jobs but didn't have time to look into it.